cmd/environment.go

```diff
@@ -74,6 +74,25 @@ const (
 	// vault namespace. The vault namespace is used if the enterprise
 	// version of Hashicorp Vault is used.
 	EnvVaultNamespace = "MINIO_SSE_VAULT_NAMESPACE"
+
+	// EnvETCDEndpoints points to the etcd backend which manages MinIO's
+	// config.json and bucket DNS SRV records.
+	EnvETCDEndpoints = "MINIO_ETCD_ENDPOINTS"
```

Comment:
This can be removed as we are not using this anymore

Reply:
removed